vscode-trace-extension/src/trace-explorer/opened-traces/trace-explorer-opened-traces-view-provider.ts

```diff
@@ -0,0 +1,128 @@
+import * as vscode from 'vscode';
+import { TraceViewerPanel } from '../../trace-viewer-panel/trace-viewer-panel';
+import { getTspClientUrl, getTraceServerUrl } from '../../utils/tspClient';
+import { signalManager, Signals } from '@trace-viewer/base/lib/signals/signal-manager';
+import { Experiment } from 'tsp-typescript-client/lib/models/experiment';
+import { OpenedTracesUpdatedSignalPayload } from '@trace-viewer/base/lib/signals/opened-traces-updated-signal-payload';
+
+export class TraceExplorerOpenedTracesViewProvider implements vscode.WebviewViewProvider {
+
+	public static readonly viewType = 'traceExplorer.openedTracesView';
```

Comment:
tab spacing, ok?

Reply:
Yeah, we should add a linter to catch that and update all classes to comment code style settings. Outside of the scope this patch. I'll update here though.

Reply:
I added a 2nd commit for eslint usage and related updates